source/concepts.tex

```diff
@@ -981,7 +981,7 @@
 \tcode{true_type}\iref{meta.type.synop},
 \tcode{int*}, and
 \tcode{bitset<N>::reference}\iref{template.bitset}
-model \exposconceptx{boolean\--testable}{boolean-testable}.
```

Comment:
Seems like this hyphenation hint is not needed anymore due to examples being a point smaller now.
![1658856382](https://user-images.githubusercontent.com/21071787/181071908-ea11afa9-964b-4db5-8506-8409e3705b2e.png)
![1658856392](https://user-images.githubusercontent.com/21071787/181071911-5d5090cc-ce8b-46b4-a755-2ab92d49c4a2.png)